test/cypress/integration/example_spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Name of the group', () => {
```

Comment:
We usually don't do underscore file names in Highcharts...

Reply:
Ah yes. Do we have a preference towards naming conventions? I see in the Cypress docs they sometimes use `example.spec.js`, but any `.js` file works so we could also omit the `spec` part.

Reply:
In `samples/unit-tests` we just use dashes. In `test/ts-node-unit-tests/test` I see that we are using dashes _and_ dots. So I guess either works.